Entity/Profile.php

```diff
@@ -1,127 +0,0 @@
-<?php
```

Comment:
Do not move this file from Entity to Model. Model is usefull when you need to extend it, but in this case, it's a BC and we don't need this feature

Reply:
What does mean "BC" acronym ? I think it is not an entity as an entity is persisted in DB then I have moved it to Model.